src/screens/Main/index.js

```diff
@@ -11,9 +10,11 @@ function Main({navigation}) {
     <SafeAreaView style={styles.container}>
       <View style={styles.containerHeader}>
         <Image source={logo} style={styles.logo} />
-        <Image source={title} style={styles.title} />
         <Text style={styles.text}>
-          Colorindo com Números - Representação de Imagens
+          COLORINDO COM NÚMEROS
+        </Text>
+        <Text style={styles.text}>
+          REPRESENTAÇÃO DE IMAGENS
```

Comment:
Esse texto é em caixa alta?

Reply:
Sim. Como no figma, mas ainda não alterei a fonte.

Reply:
No figama esse nome não é maisuculo. 
```suggestion
         Representação de Imagens
```
![Frame 231](https://user-images.githubusercontent.com/18462250/156243589-7bf9c7f3-ebb2-4c46-a0f3-6e40e764f2c6.png)

Reply:
Realmente, não tinha notado, obrigada, já alterei!

Reply:
Realmente, eu não havia notado, já alterei, obrigada.